ecf/subseasonal/stats/jevs_cfs_subseasonal_grid2grid_stats.ecf

```diff
@@ -0,0 +1,115 @@
+#PBS -q "dev"
+#PBS -A VERF-DEV
+#PBS -l walltime=02:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+
+export model=evs
+
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+
+source $HOMEevs/versions/run.ver
+
+#%include <head.h>
+#%include <envir-p1.h>
+
+############################################################
+# Load modules
+############################################################
+
+module load intel/${intel_ver}
+module load PrgEnv-intel/${PrgEnvintel_ver}
+module load craype/${craype_ver}
+module load cray-mpich/${craympich_ver}
+module load cray-pals/${craypals_ver}
+module load cfp/${cfp_ver}
+module load g2c/${g2c_ver}
+module load wgrib2/${wgrib2_ver}
+module load proj/${proj_ver}
+module load geos/${geos_ver}
+export HPC_OPT=/apps/ops/prod/libs
+module use /apps/ops/prod/libs/modulefiles/compiler/intel/${intel_ver}/
+module load intel/${intel_ver}
+module load gsl/${gsl_ver}
+module load libjpeg/${libjpeg_ver}
+module load libpng/${libpng_ver}
+module load netcdf/${netcdf_ver}
+module load udunits/${udunits_ver}
+module load nco/${nco_ver}
+module load grib_util/${grib_util_ver}
+module load prod_util/S{prod_util_ver}
+module load python/${python_ver}
+module load met/${met_ver}
+module load metplus/${metplus_ver}
+module load cdo/${cdo_ver}
+
+module list
+
+export MET_ROOT=/apps/ops/prod/libs/intel/${intel_ver}/met/${met_ver}
+export MET_BASE=${MET_ROOT}/share/met
+export PATH=${MET_ROOT}/bin:${PATH}
+export MET_bin_exec=bin
+export METviewer_AWS_scripts_dir=/lfs/h2/emc/vpppg/save/emc.vpppg/verification/metplus/metviewer_aws_scripts
+
+export RM=`which rm`
+export CUT=`which cut`
+export TR=`which tr`
+export NCAP2=`which ncap2`
+export CONVERT=`which convert`
+export NCDUMP=`which ncdump`
+
+export USER=$USER
+export DATAROOTtmp=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
+export ACCOUNT=VERF-DEV
+export QUEUE=dev
+export QUEUESHARED=dev_shared
+export QUEUESERV=dev_transfer
+export PARTITION_BATCH=
+export nproc=128
+export USE_CFP=YES
+export met_ver=${met_ver}
+export metplus_ver=${metplus_ver}
+#export cyc=%CYC%
+export cyc=00
+export NET=evs
+export evs_ver=${evs_ver}
+export STEP=stats
+export COMPONENT=subseasonal
+export RUN=atmos
+export MODELNAME=cfs
+export cfs_ver=${cfs_ver}
+#export OBSNAME=gfs
+#export gfs_ver=${gfs_ver}
+#export ccpa_ver=${ccpa_ver}
+#export obsproc_ver=${obsproc_ver}
+export VERIF_CASE=grid2grid
+
+export COMROOT=/lfs/h2/emc/vpppg/noscrub/$USER/evs_subseasonal_test/dev/com
+#export COMIN=$COMROOT/$NET/$evs_ver
+export COMIN=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMINobs=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
+export COMINclimo=$FIXevs/climos/atmos
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/evs_subseasonal_test/dev/com/$NET/$evs_ver/$STEP/$COMPONENT
+
+
```

Comment:
Add a line in the ecf script to force a desired VDATE if it will be different from the default VDATE that will be defined in the j-job file.
`export VDATE=$(date -d "today -22 day" +"%Y%m%d")`